cluster-up/cluster/kind/common.sh

```diff
@@ -253,6 +253,13 @@ function setup_kind() {
         _configure_network "$node"
     done
     prepare_config
+
+    if [[ $KUBEVIRT_DEPLOY_CDI == "true" ]]; then
```

Comment:
I take this as a workaround for the time being, while we have the kind being managed in bash and not by the go code (gocli)

Reply:
Yes, I'm not aware of any plans to move the kind providers over to gocli tbh, was that discussed?